modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala

```diff
@@ -130,7 +133,36 @@ object ObservationModel extends ObservationOptics {
         a.existence,
         a.name,
         a.status,
-        a.asterismId
+      )}
+
+  }
+
+  final case class EditPointing(
+    observationIds: List[Observation.Id],
+    asterismId:     Option[Asterism.Id],
+    targetId:       Option[Target.Id]
```

Comment:
Instead of using `Input`s here I went with the simpler `Option`.  In the case of a general edit, we have to distinguish unsetting vs ignoring pointing.  Here presumably if you explicitly edit pointing then it can be assumed you don't want to ignore it.